### ISSUE TITLE
Commands editieren für alle

### ISSUE THREAD

**Issue by dnns01** [FU-Hagen-Discord/fernuni-bot#245]
Sollte jeder Commands editieren können? 

![grafik](https://github.com/user-attachments/assets/3e68680a-93d5-496c-9f85-e8ad0b68bff2)

**Comment by Lou-M** [comment]
Ja, wobei Edits von nicht-Mod erst einem Check unterlaufen sollen, so mein Vorschlag.
